packages/backend/src/models/CatalogModel/utils/index.ts

```diff
@@ -1,34 +1,49 @@
 import {
     CatalogType,
+    DEFAULT_SPOTLIGHT_CONFIG,
     Explore,
     FieldType,
     friendlyName,
     getItemId,
     isExploreError,
+    isMetric,
     type CatalogFieldMap,
     type CatalogFieldWhere,
     type ChartFieldChanges,
     type ChartFieldUpdates,
     type ChartFieldUsageChanges,
     type ExploreError,
+    type Metric,
 } from '@lightdash/common';
 import { uniq } from 'lodash';
 import { DbCatalogIn } from '../../../database/entities/catalog';
+import { DbTag } from '../../../database/entities/tags';
+
+const getSpotlightShow = (
+    spotlight?: Explore['spotlight'] | Metric['spotlight'],
+) => {
+    const visibility =
+        spotlight?.visibility || DEFAULT_SPOTLIGHT_CONFIG.default_visibility;
+
+    return visibility === 'show';
```

Comment:
should this show be an enum ?

Reply:
We considered this but the type is already good enough for intellisense /validation